velox/common/caching/SsdFile.h

```diff
@@ -367,7 +367,13 @@ class SsdFile {
 
   /// Returns the checkpoint file path.
   std::string getCheckpointFilePath() const {
-    return fileName_ + kCheckpointExtension;
+    // Faulty file path needs to be handled manually before we switch checkpoint
```

Comment:
I think this is ok we can inject fault based on the file path match and different types of files have different suffix

Reply:
The issue here is once we enable faulty file system in the testing, the file path will have a prefix "faulty:", and since checkpoint file operations haven't been switched to velox. it will try openning a file with the prefix, resulting in failure.

Reply:
I see.